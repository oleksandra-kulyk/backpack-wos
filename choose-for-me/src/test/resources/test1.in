15 17 22
0
c3 7 REGULAR 1
c1 3 REGULAR 3
c4 8 REGULAR 1
c2 5 REGULAR 2