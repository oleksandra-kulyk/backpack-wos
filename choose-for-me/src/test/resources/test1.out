c1 3
c4 1
c2 1